### DISCUSSION TITLE
arguments for --soloCellFilter

### DISCUSSION THREAD

**Discussion by onmyojiyys** [alexdobin/STAR#1985] [Q&A]
In STAR manual document, 'EmptyDrops CR' was selected for '--soloCellFilter'. Can be followed by 10 numeric parameters: nExpectedCells maxPercentile
maxMinRatio indMin indMax umiMin umiMinFracMedian candMaxN FDR
simN. What is the specific meaning of each indicator in the 10 paramneters?

**Comment by alexdobin** [comment]
Hi @onmyojiyys 

This algorithm replicates 10X call calling function, which is based on the EmptyDrops algorithms.
If you want to tweak this algorithm, I recommend using EmptyDrops implementation.
